scripts/oes-data-migration-scripts/migration_v3.10.x_to_v3.11.py

```diff
@@ -175,31 +175,24 @@ def postingPipelineExecutionAuditEvents(auditData):
         raise e
 
 
-def getexecutionIdWithTime():
+def getexecutionIdWithTime(pipeline_executions):
     try:
         cur = audit_conn.cursor()
-        cur.execute("select audit_data -> 'executionId' as executionId from pipeline_execution_audit_events")
-        executionIdList = [item[0] for item in cur.fetchall()]
-        count = 0
-        if executionIdList != None:
-            for executionId in executionIdList:
-                logging.info("Updating pipeline execution time for execution Id: "+ str(executionId))
-                cur.execute(
-                    "select data -> 'details' ->>'created' as executionTime from audit_events where data -> 'content' ->> 'executionId' ='" + executionId + "'")
-                dateDetails = [executionTime[0] for executionTime in cur.fetchall()]
-                if dateDetails != None:
-                    count += 1
-                    dateDetails = str(dateDetails[0])
-                    updatedTime_date_time = datetime.datetime.utcfromtimestamp(int(dateDetails) / 1000)
-                    val = """UPDATE pipeline_execution_audit_events SET created_at ={}, updated_at ={} WHERE audit_data ->> 'executionId' = {}""".format(
-                        '\'' + str(updatedTime_date_time) + '\'', '\'' + str(updatedTime_date_time) + '\'',
-                        '\'' + executionId + '\'')
-                    cur.execute(val)
-        logging.info("Total updated created time for pipeline execution Data: " + str(count))
-        print("Total updated created time for pipeline execution Data: " + str(count))
+        for pipeline_execution in pipeline_executions:
+            executionId = pipeline_execution[1]
+            dateDetails = str(pipeline_execution[2])
+            updatedTime_date_time = str(datetime.datetime.utcfromtimestamp(int(dateDetails) / 1000))
+            logging.info(f"Updating execution : {executionId}")
```

Comment:
@Pranav-b-7 not compolsury but along with execution id we could log the created time ( updatedTime_date_time) ! leaving on you